apps/src/templates/codeReviewGroups/CodeReviewGroupsLoader.jsx

```diff
@@ -11,7 +11,7 @@ export default function CodeReviewGroupsLoader({sectionId}) {
         const api = new CodeReviewGroupsDataApi(sectionId);
         api
           .getCodeReviewGroups()
-          .then(groups => onLoadSuccess([convertGroupData(groups)]))
+          .then(groups => onLoadSuccess([groups]))
```

Comment:
Yep that's because `onLoadSuccess` needs to be called with an array of parameters. This array of parameters will be passed to the render function as a set of arguments. For this function, there's only one parameter (the array groups) so it's a single member in an array.